protocols/isis/packet/csnp.go

```diff
@@ -97,7 +101,7 @@ func NewCSNPs(sourceID types.SourceID, lspEntries []*LSPEntry, maxPDULen int) []
 func newCSNP(sourceID types.SourceID, startLSPID LSPID, endLSPID LSPID, tlvs []TLV) *CSNP {
 	tlvsLen := uint16(0)
 	for i := range tlvs {
-		tlvsLen += uint16(tlvs[i].Length())
+		tlvsLen += uint16(tlvs[i].Length()) + 2
```

Comment:
A constant with an explainatory name would improve the readabiltiy here (magic figure)

Reply:
Agreed. Or a comment at least

Reply:
Fixed.